src/document.rs

```diff
@@ -47,6 +47,38 @@ fn yaml_to_liquid(yaml: &Yaml) -> Option<Value> {
     }
 }
 
+/// Escape HTML string
+///
+/// The function has been extracted from `librustdoc/html/escape.rs`.
+/// See also http://stackoverflow.com/questions/7381974 for details.
+pub fn escape_html(html: &str) -> String {
+    let mut escaped = String::new();
+    let mut last = 0;
+    for (i, ch) in html.bytes().enumerate() {
+        match ch as char {
+            '<' | '>' | '&' | '\'' | '"' => {
+                escaped.push_str(&html[last..i]);
+                let s = match ch as char {
+                    '>' => "&gt;",
+                    '<' => "&lt;",
+                    '&' => "&amp;",
+                    '\'' => "&#39;",
+                    '"' => "&quot;",
+                    _ => unreachable!(),
+                };
+                escaped.push_str(s);
+                last = i + 1;
+            }
+            _ => {}
+        }
+    }
+
+    if last < html.len() {
+        escaped.push_str(&html[last..]);
+    }
+    escaped
+}
```

Comment:
i was confused by my rss reader when i was testing this locally - in fact rss crate escapes html for us: https://github.com/rust-syndication/rss/blob/0.3.1/src/item.rs#L55 (this fact is also covered by `tests/target/rss/rss.xml`)

so i removed `escape_html` function